### ISSUE TITLE
Add exception to `property-no-vendor-prefix` for `-webkit-background-size` when it only has one value

### ISSUE THREAD

**Issue by Mouvedia** [stylelint/stylelint#7727]
### What minimal example or steps are needed to reproduce the bug?

```css
a {
  -ms-interpolation-mode: bicubic;
  -webkit-background-size: 1px;
}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "property-no-vendor-prefix": true
  }
}
```

### Do you have a proposal to fix the bug?

1. remove this exception:

https://github.com/stylelint/stylelint/blob/d902c0b7cc1233b74cc4959eff864cb3e2d87f73/lib/utils/isAutoprefixable.mjs#L342-L346

2. remove this property:

https://github.com/stylelint/stylelint/blob/d902c0b7cc1233b74cc4959eff864cb3e2d87f73/lib/utils/isAutoprefixable.mjs#L78

### References

the discussion starting with https://github.com/stylelint/stylelint/issues/7294#issuecomment-1793474109
whatwg/compat#28
w3c/csswg-drafts#7802

**Comment by ybiquitous** [comment]
@Mouvedia Thanks for the issue. To help us understand the issue, I've created a minimum demo:
[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcICGACYAdAdilBaAtgM64CWmALjAE4AOEANkuSRJgdDPCgEYlgCuvMAG4sOXAHcY3ANYlyubkjAyA5lQj9MUXIRIAvTigCMNAB6jMAXxAAaEADMS9GADkk+OIhhmPNFwB0YISEduCsTqoIIBjYKOggVPwuhAlcsTg4CTQaNNTkAJ64mBC4AG4w2hBUuDkwTmZpKORJMGIoVlg29pCYkQBi1fhM0QBWhKxhsDShiBnxIISFLvRk5E0JADzGAOwBAAwHCV0gVkA)

```css
a {
  -ms-interpolation-mode: bicubic;
  -webkit-background-size: 1px;
}

/* ↓ autofixed ↓ */

a {
  interpolation-mode: bicubic;
  background-size: 1px;
}
```

According to the new spec, is this autofix now invalid, right?

**Comment by Mouvedia** [comment]
> According to the new spec, this autofix is now invalid, right?

Not exactly. You should read the reference links that I have provided for the details.
tl;dr: having the exception was an oversight on our part and unprefixing `-vendor-background-size` is not safe

**Comment by ybiquitous** [comment]
Okay. Let's address this issue for v17. 👍🏼

**Comment by Mouvedia** [comment]
Thinking about it more, this can be fixed without being a breaking change.
